.travis.yml

```diff
@@ -3,7 +3,7 @@
 language: julia
 
 julia:
-  - 1 # latest 1.x.y
+  - 1.6 # latest 1.x.y
```

Comment:
Why did you change this?

Reply:
Because `P4est_jll.jl@2.3.1` [requires Julia 1.6](https://github.com/JuliaBinaryWrappers/P4est_jll.jl/blob/c281fee10ffd8f172859055b4ea9c6b5f26bc077/Project.toml#L14) and this seemed to be the underlying cause for troubles once I started pinning the JLL version to v2.3.1.